cookietemple/lint/template_linter.py

```diff
@@ -329,6 +330,16 @@ def _strip_ansi_codes(self, string, replace_with=""):
 
         return ansi_escape.sub(replace_with, string)
 
+    @staticmethod
+    def get_calling_class() -> str:
+        """
+        Get the name of the linting class, calling the linter
+
+        :return: The name of the linting class calling
+        """
+        stack = inspect.stack()
+        return stack[1][0].f_locals["self"].__class__.__name__
+
```

Comment:
There are tons of other ways to accomplish this, one would be using `locals()`.

Reply:
Honestly, this is not the most readable code haha